### PR TITLE
feat: Add response time service

### PR DESCRIPTION
**Pull request by ebebbington**

### CODE REVIEW

File: src/services/response_time/response_time.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { IService, Request, Response, Service } from "../../../mod.ts";
+
+export class ResponseTimeService extends Service implements IService {
+  #startTime = 0;
+
+  #endTime = 0;
+
+  constructor() {
+    super();
+  }
```
Comment (Guergeiro) [inline src/services/response_time/response_time.ts:10]: Why constructor?

Comment (ebebbington) [reply src/services/response_time/response_time.ts:10]: No reason really, just didn’t realise until now how pointless it was :p good shout, I’ll remove it

Comment (ebebbington) [reply src/services/response_time/response_time.ts]: done :) https://github.com/drashland/drash/pull/567/commits/23be120b70398ca994ebd9d28f84cca4c704e7d7

**Review by Guergeiro — APPROVED** [review]

**Comment by ebebbington** [comment]
docs are https://github.com/drashland/website-v2/pull/41

**Comment by crookse** [comment]
FAST AF BOIIIII

**Pull request merged**